### PR TITLE
chore: replace Travis with GitHub actions

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

**Comment by targos** [comment]
The tests pass locally. I wonder what's different in CI? @fraxken

**Comment by targos** [comment]
I fixed it by sorting the files array. Apparently the system that runs GitHub actions doesn't return them in the expected order.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: master
```
Comment (Mesteery) [inline .github/workflows/nodejs.yml:5]:
```suggestion
    branches: $default-branch
```

Comment (targos) [reply .github/workflows/nodejs.yml:5]: `$default-branch` is not a valid string here. It's just a placeholder that you can put in templates and that gets replaced by github when you add the template to a repo.

**Pull request merged**